Assets/Scripts/Control&Input/PlayerInputManagerController.cs

```diff
@@ -81,6 +82,8 @@ private void OnPlayerJoined(PlayerInput playerInput)
         inputManager.PlayerCamera.enabled = false;
         LocalPlayerInputs.Add(inputManager);
         onPlayerInputJoined?.Invoke(inputManager);
+        if (inputManager.IsMouseAndKeyboard)
+            Cursor.visible = true;
```

Comment:
but why visible?

Reply:
Cursor is hidden unless we absolutely need to show the mouse (we have a keyboard). We don't need to do this as is (since we can't click buttons with the mouse, but I really want to implement that soon.)

Reply:
But it's still invisible now I hope? Can't really test if it appears in loading screens etc. since this branch doesn't have the compilation error fix yet. If it does, please just _actually_ hide it and add a TODO comment for showing the mouse again 😅